test/templates/message-broker-with-entities-2nd-call/src/main/resources/config/application.yml

```diff
@@ -161,6 +161,12 @@ jhipster:
 
 kafka:
   bootstrap.servers: ${KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+  producer:
+    foo:
+      name: queuing.message_broker_with_entities.foo
+      enabled: true
+      '[key.serializer]': org.apache.kafka.common.serialization.StringSerializer
+      '[value.serializer]': com.mycompany.myapp.service.kafka.serializer.FooSerializer
   polling.timeout: 10000
   consumer:
```

Comment:
The order should be `bootstrap.servers`, then `polling.timeout` and finally `consumer:` and `producer:`

Reply:
Yes I put intentionnaly in this order(given part of test) to assert that the code rearrange it

Reply:
I see!